spec/defines/seboolean_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+describe 'types::selboolean' do
+  let(:title) { 'nfs_export_all_ro' }
+
+  context 'selboolean with all options specified' do
+    let(:params) do
+      { :persistent => true,
+        :value      => 'on',
+      }
+    end
+
+    it {
+      should contain_selboolean('nfs_export_all_ro').with({
+        'persistent' => true,
+        'value'      => 'on',
+      })
+    }
+  end
+
+  context 'selboolean with min options specified - on' do
```

Comment:
you could do these two in a for loop

https://github.com/ghoneycutt/puppet-module-types/blob/master/spec/defines/file_spec.rb#L72-86